src/main/java/org/owasp/esapi/util/ObjFactory.java

```diff
@@ -30,11 +30,11 @@
  * 		...
  * 		// Typically these would be populated from some Java properties file
  * 		String barName = "com.example.foo.Bar";
- * 		String beerBrand = "com.example.brewery.Guiness";
+ * 		String beerBrand = "com.example.brewery.Guinness";
```

Comment:
You are cracking me up this morning!

Reply:
@xeno6696 So, misspelled, and apparently ignored until I finally fix it. Uh huh. You guys are missing all my good jokes. Did you see the one on line 111 of ObjFactoryTest.java. That's been there for years as well and no one has said anything...probably because everyone agrees. :)